specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-05-02-preview/managedClusters.json

```diff
@@ -3255,6 +3255,7 @@
           "description": "Both patch version <major.minor.patch> and <major.minor> are supported. When <major.minor> is specified, the latest supported patch version is chosen automatically. Updating the agent pool with the same <major.minor> once it has been created will not trigger an upgrade, even if a newer patch version is available. As a best practice, you should upgrade all node pools in an AKS cluster to the same Kubernetes version. The node pool version must have the same major version as the control plane. The node pool minor version must be within two minor versions of the control plane version. The node pool version cannot be greater than the control plane version. For more information see [upgrading a node pool](https://docs.microsoft.com/azure/aks/use-multiple-node-pools#upgrade-a-node-pool)."
         },
         "currentOrchestratorVersion": {
+          "readOnly": true,
```

Comment:
Please only modify this swagger file (05-02-preview) in this PR. And go through the breaking change review in PR targets the main branch.

Reply:
> Please only modify this swagger file (05-02-preview) in this PR. And go through the breaking change review in PR targets the main branch.

Got it, changes to other two files have been reverted.